app/controllers/partners/base_controller.rb

```diff
@@ -10,5 +10,32 @@ class BaseController < ApplicationController
     def redirect_to_root
       redirect_to root_path
     end
+
+    helper_method :current_partner
+    def current_partner
+      current_partner_user.partner
+    end
+
+    helper_method :valid_items
+    def valid_items
+      # @valid_items ||= DiaperBankClient.get_available_items(current_partner.diaper_bank_id)
+      @valid_items ||= current_partner.organization.valid_items
+    end
```

Comment:
Instead of using a helper method to retrieve valid_items, I feel like it is better to load them into the controller. It might be duplicated but I think keeping this code with where we need it makes sense. Like how it is done here:
https://github.com/rubyforgood/diaper/blob/b73c315cab92b80e58845d2ce85e1a3f6db10fba/app/controllers/partners/requests_controller.rb#L17

Reply:
Made that change. It turns out we don't even need to set the instance variable `@valid_items` in any of the controllers if we just do `current_partner.organization.valid_items` in the views. LMK if you think that's going too far with inlining

Reply:
Oh nm I forgot I am memoizing it in one place where we use the hash to find the item name more than once in the view